src/puppetlabs/puppetdb/facts.clj

```diff
@@ -69,7 +69,7 @@
 (defn unescape-string
   "Strip escaped quotes from a string."
   [^String s]
-  (if (= \" (.charAt s 0))
+  (if (= \" (first s) (last s))
```

Comment:
Do we not want to strip escaped quotes on a single side as well? "\"foo" -> "foo"